backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/export/SVGExportService.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.diagrams.export;
+
+import java.util.Objects;
+
+import org.eclipse.sirius.web.diagrams.Diagram;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service used to export a diagram to SVG.
+ *
+ * @author rpage
+ */
+@Service
+public class SVGExportService {
+    private final SVGExportContext processor;
```

Comment:
- This context concept is called processor everywhere, it does not make sense
- I don't understand why you can't call the DiagramExportHandler here